packages/common/src/helpers/__tests__/errors.spec.ts

```diff
@@ -0,0 +1,204 @@
+/*
+ * Copyright (c) 2018-2021 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+
+import axios, { AxiosError, AxiosResponse } from 'axios';
+import { HttpError } from '@kubernetes/client-node';
+import AxiosMockAdapter from 'axios-mock-adapter';
+import * as http from 'http';
+import { getMessage } from '../errors';
+
+let mockAxios = new AxiosMockAdapter(axios);
+
+describe('Errors helper', () => {
+
+  // mute the outputs
+  console.error = jest.fn();
+
+  afterEach(() => {
+    jest.clearAllMocks();
+    mockAxios.resetHandlers();
+  });
+
+  it('should return default message', () => {
+    expect(getMessage(undefined)).toEqual('Unexpected error.');
+  });
+
```

Comment:
It should support the type 'string' too.

```
 it('should return error message', () => {
    const message = 'Expected error message.';
    expect(getMessage(message)).toEqual(message);
  });
```

Reply:
And have test coverage for  'isError' method:

```
  it('should check if it is an error or not', () => {
    const message = 'Expected error.';
    const error = new Error(message);
    expect(isError(error)).toEqual(true);
    expect(isError(message)).toEqual(false);
  });

```

Reply:
I'll add these test cases in the following commit.